File: 2022.01.22/python/1427.py
s = input()
result = ""
cnt = [0 for _ in range(10)]
for char in s:
    cnt[int(char)] += 1

for i in range(9,-1,-1):
    while cnt[i]:
        cnt[i] -= 1
        result += str(i)
print(result)

File: Boj.단계별/12_정렬/javascript/1427.js
const s = require('fs').readFileSync('/dev/stdin').toString().split('\n')[0];
const cnt = s.split('').reduce((array,cur) => {
  array[cur]++;
  return array;
},new Array(10).fill(0))
let result = "";

for (let i = 9 ; i>= 0 ;i--){
  while (cnt[i]--) result += i;
}
console.log(result)